FlightReservation/components/screens/Login.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {View, ScrollView, Text, TextInput, Button, Linking} from 'react-native';
+import RegisterStyles from '../assets/styles/RegisterStyles';
+
+const Login = () => {
+  return (
+    <ScrollView style={RegisterStyles.container}>
+      <View style={RegisterStyles.group}>
+        <Text style={RegisterStyles.lblTitle}>Log In</Text>
+        <View style={RegisterStyles.row}>
+          <Text style={RegisterStyles.lblLabelEmailPass}>Email</Text>
+          <Text style={RegisterStyles.lblErrorEmailPass}>
+            *Email in use. Use a different email
+          </Text>
+        </View>
+        <TextInput style={RegisterStyles.txbInputText} />
+        <View style={RegisterStyles.row}>
+          <Text style={RegisterStyles.lblLabelEmailPass}>Password</Text>
+          <Text style={RegisterStyles.lblErrorEmailPass}>
+            *Incorrect email and/or password
+          </Text>
+        </View>
+        <TextInput secureTextEntry style={RegisterStyles.txbInputText} />
+        <Text style={RegisterStyles.lblPasswordIndication}>
+          Use 8 or more characters with a mix of letters, numbers, {'\n'}and
+          symbols
+        </Text>
```

Comment:
Personally, I don't see that it's necessary to have this label on the LogIn screen, generally what is set for the password for these types of screens is "_Forgot your password?_", but it's only a comment. Everything is really nice! 🔥